website/source/docs/providers/chef/r/data_bag.html.markdown

```diff
@@ -6,7 +6,7 @@ description: |-
   Creates and manages a data bag in Chef Server.
 ---
 
-# chef\_data\_bag
+# chef_data_bag
```

Comment:
Seem so 
![screen shot 2017-04-10 at 11 36 02 am](https://cloud.githubusercontent.com/assets/61721/24872139/ebe4aede-1de1-11e7-86ed-2737ef9de4c4.png)

I welcome this change 😄